platform/security/security-util/src/test/java/org/codice/ddf/security/util/SAMLUtilsTest.java

```diff
@@ -0,0 +1,119 @@
+
+  @Test
+  public void testSAMLAssertionParse() throws SecurityServiceException {
+
+    SecurityToken securityToken = SAML_UTILS.getSecurityTokenFromSAMLAssertion(SAML_ASSERTION);
+    assertThat(securityToken, is(instanceOf(SecurityToken.class)));
+  }
+
+  @Test
+  public void testSAMLAssertionParseFail() throws SecurityServiceException {
+
+    SecurityToken securityToken = SAML_UTILS.getSecurityTokenFromSAMLAssertion(BAD_SAML_ASSERTION);
```

Comment:
❓This test is identical to the previous test except for the test string. Is there any way to verify that the parser failed?